pyproject.toml

```
@@ -111,36 +123,21 @@ dev = [
     "docutils",
 ]
 # Algorithms
-algorithms = [  # tested algorithms
-    "scanpy[leiden]",
-    "scanpy[magic]",
-    "scanpy[skmisc]",
-    "scanpy[harmony]",
-    "scanpy[scanorama]",
-    "scanpy[scrublet]",
-]
-algorithms-full = [
-    "scanpy[algorithms]",
-    "scanpy[louvain]",
-    "scanpy[bbknn]",
-]
+paga = ["python-igraph"]
 louvain = ["python-igraph", "louvain>=0.6,!=0.6.2"]  # Louvain community detection
 leiden = ["python-igraph", "leidenalg"]  # Leiden community detection
 bbknn = ["bbknn"]  # Batch balanced KNN (batch correction)
 magic = ["magic-impute>=2.0"]  # MAGIC imputation method
-skmisc = ["scikit-misc>=0.1.3"]  # highly_variable_genes method 'seurat_v3'
+seurat = ["scikit-misc>=0.1.3"]  # highly_variable_genes method 'seurat_v3'
```

```suggestion
skmisc = ["scikit-misc>=0.1.3"]  # highly_variable_genes method 'seurat_v3'
```

I feel like this name introduces more confusion than it solves.